src/commands/dockersupport/localDockerSupport.ts

```diff
@@ -14,51 +14,43 @@ import { validateDockerInstalled } from "./validateDockerInstalled";
-        if (devContainerName) {
-            // check if the function app is in Linux
-            if (node.site.reserved) {
-                // asks if the user wants to use Docker for initializing the project locally
-                const useDocker: string = await prompt(context);
-                if (useDocker === "yes") {
-                    // download dev container
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/devcontainer.json`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'devcontainer.json').fsPath
-                    );
-                    // download docker file
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/Dockerfile`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'Dockerfile').fsPath
-                    );
-                    // external - check if Docker is installed, Remote Development extension AND Docker Extension
-                    if (!validateDockerInstalled()) {
-                        // if Docker is not downloaded - ask the user if they'd like to download Docker
-                        const downloadDocker: string = await prompt2(context);
-                        if (downloadDocker === "yes") {
-                            // Check if Operating System is Windows
-                            if (process.platform == "win32") {
-                                // Download Docker with an MSI package
-                            } else {
-                                // Not windows: display link to download docker externally from Docker documentation
-                                void vscode.window.showInformationMessage(localize('downloadDocker', 'Check the (Docker documentation)[https://docs.docker.com/get-docker/] to download Docker for your system'));
-                            }
-                        } else {
-                            void vscode.window.showInformationMessage(localize('noDocker', 'Continuing without the use of Docker as user requested'));
-                        }
+    if (node && devContainerName && node.site.reserved) {
+        // asks if the user wants to use Docker for initializing the project locally
+        const useDocker: string = await prompt(context);
+        if (useDocker === "yes") {
+            await downloadLocalDevFiles(devContainerFolderPathUri, devContainerName);
+            // external - check if Docker is installed, Remote Development extension AND Docker Extension
+            if (!validateDockerInstalled()) {
+                // if Docker is not downloaded - ask the user if they'd like to download Docker
+                if (await prompt2(context) === "yes") {
```

Comment:
we should try our best to always avoid having inline strings in the code. some reasons for this:
1. higher chances of typos
2. higher chances of runtime failures due to typos.

The best way to handle these is to either have class level or project level constants.
so at the top you can have something like:

```
export const PROMPT_YES = 'yes'
export const PLATFORM_WIN32 = 'win32'
```
and than reference that down in this function:

`process.platform.ToLower() === PLATFORM_WIN32`

Also note the case sensitivity.

Reply:
That makes sense, will add that change